addons/actions/src/models/ActionDisplay.ts

```diff
@@ -0,0 +1,11 @@
+import { ActionOptions } from './ActionOptions';
+
+export interface ActionDisplay {
+  id: string;
+  data: {
+    name: string;
+    args: any;
```

Comment:
I think you can make this stricter to `any[]` or not?

Reply:
I didn't find any clue in addon codebase to know if it's necessary an array or not so for now I will just put `any`.